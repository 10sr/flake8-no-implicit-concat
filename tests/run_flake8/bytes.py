a = b"aaa" b"bbb"
b = [b"aaa",
     b"bbb"
     b"ccc"]
c = rb"aaa" b"bbb"
