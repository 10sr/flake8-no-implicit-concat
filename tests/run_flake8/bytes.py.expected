./bytes.py:1:11: NIC101 Implicitly concatenated bytes literals on one line
./bytes.py:3:12: NIC102 Implicitly concatenated bytes literals over multiple lines
./bytes.py:5:12: NIC101 Implicitly concatenated bytes literals on one line
